bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -171,22 +188,22 @@ It's possible to enter a value manually or set `shutterRun=AUTO` (default) to ca
 
 BUS gateway and things configuration:
 
-```xtend
+```json
```

Comment:
Please remove this at all. The type doesn't need to be specified any longer.

Reply:
removed these type tags